app/bundles/CategoryBundle/Model/CategoryModel.php

```diff
@@ -186,4 +187,36 @@ public function getLookupResults($bundle, $filter = '', $limit = 10): array
 
         return $this->categoriesByBundleCache[$key] = $this->getRepository()->getCategoryList($bundle, $filter, $limit, 0);
     }
+
+    /**
+     * @return array<mixed>
+     */
+    public function getUsage(Category $category): array
+    {
+        $bundle = $category->getBundle();
+
+        $types = [];
+        if ($this->dispatcher->hasListeners(CategoryEvents::CATEGORY_TYPE_ENTITY)) {
+            $event = $this->dispatcher->dispatch(new CategoryTypeEntityEvent(), CategoryEvents::CATEGORY_TYPE_ENTITY);
+            $types = $event->getCategoryTypeEntity($bundle);
+        }
+
+        $data = [];
+        foreach ($types as $type) {
+            /** @phpstan-var class-string $class */
+            $class     = $type['class'];
```

Comment:
Can we say in the return type of the `getCategoryTypeEntity` that the returned array is type of `@return class-string[]`? This way this comment could be omited.